src/main/scala/io/keen/client/scala/Logger.scala

```diff
@@ -0,0 +1,56 @@
+package io.keen.client.scala
+
+import grizzled.slf4j.Logging
+
+object Logger extends Logging {
```

Comment:
(I'm also a bit rusty on the whole SLF4J "binding" thing, excuse me if there's an obvious solution to the point I'm raising as-is—providing documentation of how to configure it for forgetful users like me would be a good inclusion :smile:).